### PR TITLE
Update/api

### PR DESCRIPTION
**Pull request by Hamdrive**
## Overview

Successfully migrated from Netlify serverless, and now accessing serverless function from Vercel, giving faster response and ease of setup and maintenance

Changes:
1. Migrate serverless function from Netlify to Vercel
2. Refactor response handling accordingly
3. Fixes to UI and other styles as per reviewers

### CODE REVIEW

File: api/getTwitterUser.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        Authorization: process.env.REACT_APP_TWITTER_BEARER_TOKEN,
+      },
+    });
+
+    if (!response.ok) {
+      res.status(response.status).json(response.statusText);
+    } else {
+      const data = await response.json();
+
+      return res.status(200).json(data);
```
Comment (shivampandey0) [inline api/getTwitterUser.js]: no need for `return` here

File: api/getTwitterUserTweets.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        Authorization: process.env.REACT_APP_TWITTER_BEARER_TOKEN,
+      },
+    });
+
+    if (!response.ok) {
+      res.status(response.status).json(response.statusText);
+    } else {
+      const data = await response.json();
+
+      return res.status(200).json(data);
```
Comment (shivampandey0) [inline api/getTwitterUserTweets.js]: here too

**Review by shivampandey0 — APPROVED** [review]
Good work with migrating serverless functions to vercel.

good to merge

**Review by abhij1607 — COMMENTED** [review]
Very clean and readable code
good class naming followed
Can be merged

**Pull request merged**